src/main/java/org/usfirst/frc4904/auton/PickupAndShootCenter.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc4904.auton;
+
+import java.util.Arrays;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.FlywheelSpinDown;
+import org.usfirst.frc4904.robot.commands.RunIntake;
+import org.usfirst.frc4904.robot.commands.Shoot;
+import org.usfirst.frc4904.standard.commands.chassis.SimpleSplines;
+
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+
+class PickupAndShootCenter extends AutonRoutine {
+
+        public PickupAndShootCenter() {
```

Comment:
maybe clarify that this is `PickupAndShootCenter`, where we don't reevaluate our position.

Reply:
you mean that it doesn't use localization?